src/available_services.html

```diff
@@ -11,31 +11,54 @@
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Maven+Pro&display=swap" rel="stylesheet">
 <body>
     <div class="banner">
         Harry Robbins, CPA
     </div>
     <include src="includes/navigation.html"></include>
-    <div class="servicesBox">
-        <div class="services">
-            <h1>Available services:</h1>
-            <ul>
-                <li>Personal, Business, Estate, Trust, Payroll andSales Tax Planning and Assitance.</li>
-                <li>Business Planning to Include Budgeting and Projections.</li>
-                <li>Tax Audit Representaion.</li>
-                <li>Choosing and Selecting Choice of Business Entity, LLC, S Corporation, C Corporation, Partnership and
-                    Sole Proprietorship.</li>
-                <li>Estate and Financial Planning.</li>
-                <li>Problem Business Analysis.</li>
-                <li>Cash Flow Analysis.</li>
-                <li>Cost Accounting.</li>
-                <li>Software Selection.</li>
-                <li>Bookkeeping and Payroll Assistance.</li>
-                <li>Assistance With Governmental Agencies.</li>
-                <li>Out of State Tax Filings.</li>
-                <li>Contract CFO and Controllership.</li>
-            </ul>
+    <header class="services_heading">Available Services</header>
+    <div class="services_box">
+        <div class="service_card_container">
+            <img src="images/stock_photo_taxes.jpg" alt="money" class="services_picture" />
+            <div class="card_details">
+                <div class="service_category">Tax Services and Preparation</div>
+                <ul class="service_lists">
+                    <li>Personal, Business, Estate, Trust, Payroll and Sales Tax Planning, Assistance and Preparation
+                    </li>
+                    <li>Out of State Tax Filings</li>
+                    <li>Tax Audit Representation</li>
+                </ul>
+            </div>
+        </div>
+        <div class="service_card_container">
+            <img src="images/stock_photo_lightbulb.jpg" alt="consulting" class="services_picture" />
+            <div class="card_details">
+                <div class="service_category">Business Solutions</div>
+                <ul class="service_lists">
+                    <li>Choosing and selecting choice of Business Entity, LLC, S Corporation, C Corporation, Partnership
+                        and Sole Proprietorship</li>
+                    <li>Problem Business Analysis</li>
+                    <li>Cash Flow Analysis and Cost Accounting</li>
+                    <li>Accounting Software Selection</li>
+                    <li>Bookkeeping and Payroll Assistance</li>
+                </ul>
+            </div>
+        </div>
+        <div class="service_card_container">
+            <img src="images/stock_photo_consultation.jpg" alt="computer_desk" class="services_picture" />
```

Comment:
keep your css prefix consistent for styles that belong together. all the others are `.service_<foobar>`
This one is `.services_` (plural). Make this one singular also.